### PR TITLE
Created new buttons in the settings to reset color and toggle cursor 

### PR DESCRIPTION
**Pull request by RJMaradiaga**
changes for creating the three buttons (reset sky color, reset floor, color, toggle cursor). The most important change is the toggle cursor function so that users can take pictures of their creation without seeing it. 

The old menu: 
image

New menu: 
image

### CODE REVIEW

**Review by davidjonesma — COMMENTED** [review]
All the changes good! no issues found with code, but not sure what's keeping it from passing checks

**Comment by bryanmontalvan** [comment]
```
@@ -10,10 +10,11 @@
          "defaultLight": true,
          "floorColor": "#222",
          "lightIndicator": false,
          "moveSpeed": 150,
          "showCoordHelper": true,
    +     "showCursor": true,
          "showFloor": true,
          "skyColor": "white",
          "viewOnly": false,
        },
        "ts": 0,
      [34](https://github.com/engaging-computing/MYR/runs/7308417303?check_suite_focus=true#step:8:35)4 |
      3[45](https://github.com/engaging-computing/MYR/runs/7308417303?check_suite_focus=true#step:8:46) |     it("should return the initial state", () => {
    > 3[46](https://github.com/engaging-computing/MYR/runs/7308417303?check_suite_focus=true#step:8:47) |         expect(scene(undefined, {})).toEqual(creatDefaultScene());
          |                                      ^
      3[47](https://github.com/engaging-computing/MYR/runs/7308417303?check_suite_focus=true#step:8:48) |     });
      3[48](https://github.com/engaging-computing/MYR/runs/7308417303?check_suite_focus=true#step:8:49) |
      3[49](https://github.com/engaging-computing/MYR/runs/7308417303?check_suite_focus=true#step:8:50) |     it("should NAME_SCENE", () => {
 ```

@RJMaradiaga The checks fail because one of the unit tests is failing. It seems to be the `Scene Render` test in `src/tests/App.test,js`. You will have to add these two new buttons and their state to the `return {` field

File: src/components/structural/header/SceneConfigMenu.js [hunk 1]
```diff
@@ -423,6 +424,29 @@ class ConfigModal extends Component {
...
+        return (
+            <ButtonBase
+                style={style}
+                onClick={() => {
+                    this.props.handleRender();
+                    this.props.sceneActions.toggleCursor();
+                }} >
+                {
+                    this.props.scene.settings.showCursor
+                        ? <Icon className="material-icons">toggle_on</Icon>
```
Comment (bryanmontalvan) [inline src/components/structural/header/SceneConfigMenu.js:442]: Could you change the icon to something that represents the action of "resetting" or "clearing"?

Example: https://mui.com/material-ui/material-icons/?query=restart&selected=RestartAlt

**Review by jf-kelly — APPROVED** [review]
I like these changes, I'd say they're good to go

**Pull request merged**